### PR TITLE
[SDK-1429] Release Push Templates v1.0.2

### PR DESCRIPTION
**Pull request by william-ct**
- UI bug fixes for Push Notification metadata
- Timer template fixes for `title`, `title_alt` along with `pt_big_img`, `pt_big_img_alt`
- Small content view text cropping fixes for Android versions less than 23
- Darker arrows for navigation in ManualCarousel & Filmstrip Carousel template
- Fixes five icon template dismissible on multiple onClickListeners
- Fixes UI issue when only 3 icons were configured in Five icon template
- Fix validation for `deepLink(optional)` key in ZeroBezel template

### CODE REVIEW

File: clevertap-pushtemplates/src/main/java/com/clevertap/android/pushtemplates/TemplateRenderer.kt [hunk 1]
```diff
@@ -215,6 +236,47 @@ class TemplateRenderer : INotificationRenderer {
...
+                    /**
+                     *  Update existing payload bundle with new title,msg,img for Basic template
+                     */
+                    val ptJsonStr  = basicTemplateBundle.getString(PTConstants.PT_JSON)
+                    var ptJsonObj: JSONObject? = null
+                    if (ptJsonStr != null) {
+                        try {
+                            ptJsonObj = JSONObject(ptJsonStr)
+                        } catch (e: Exception) {
+                            System.err.println("Unable to convert JSON to String")
```
Comment (william-ct) [inline clevertap-pushtemplates/src/main/java/com/clevertap/android/pushtemplates/TemplateRenderer.kt:250]: Add Logger-verbose

File: clevertap-pushtemplates/src/main/java/com/clevertap/android/pushtemplates/styles/Style.kt [hunk 2]
```diff
@@ -8,6 +8,7 @@ import android.text.Html
 import android.widget.RemoteViews
 import androidx.core.app.NotificationCompat
 import com.clevertap.android.pushtemplates.TemplateRenderer
+import com.clevertap.android.pushtemplates.content.ContentView
```
Comment (william-ct) [inline clevertap-pushtemplates/src/main/java/com/clevertap/android/pushtemplates/styles/Style.kt:11]: remove unused import

File: templates/CTPUSHTEMPLATESCHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,14 @@
 ## CleverTap Push Templates SDK CHANGE LOG
 
+### Version 1.0.1 (April 26, 2022)
```
Comment (william-ct) [inline templates/CTPUSHTEMPLATESCHANGELOG.md:3]: Change version

File: clevertap-pushtemplates/src/main/java/com/clevertap/android/pushtemplates/TemplateRenderer.kt (hunk 1 above)
Comment (william-ct) [reply clevertap-pushtemplates/src/main/java/com/clevertap/android/pushtemplates/TemplateRenderer.kt]: Done

**Review by piyush-kukadiya — APPROVED** [review]
LGTM

**Pull request merged**